test/csuite/wt8246_compact_rts_data_correctness/main.c

```diff
@@ -401,8 +398,7 @@ remove_records(WT_SESSION *session, const char *uri, int commit_ts)
     testutil_check(session->open_cursor(session, uri, NULL, NULL, &cursor));
 
     testutil_snprintf(tscfg, sizeof(tscfg), "commit_timestamp=%d", commit_ts);
-    /* Remove 1/3 of the records from the middle of the key range. */
-    for (i = NUM_RECORDS / 3; i < (NUM_RECORDS * 2) / 2; i++) {
+    for (i = 1; i <= NUM_RECORDS / 3; i++) {
```

Comment:
@etienneptl Does it matter if the blocks are removed from the middle of the key range or from the starting?

Reply:
Yes, it does. If we have holes at the start and data at the end of the file, compaction will try to move the data towards the beginning of the file.
If you have the data at the start of the file and a hole at the end, compact won't have much work to do, if none. It might only be able to move the checkpoint information.

Reply:
@etienneptl Yeah I agree, As discussed holes at the start of the file are more reliable for compact.